src/script/ceph-release-notes

```diff
@@ -138,21 +139,19 @@ def make_release_notes(gh, repo, ref, plaintext, verbose, strict, use_tags):
                     line = line.strip()
                     if line:
                         lines.append(line)
+                if len(lines) == 0:
+                    continue
                 duplicates_pr_title = lines[0] == pr['title'].strip()
                 if duplicates_pr_title:
-                    lines.pop(0)
-                if len(lines) == 0:
-                    if duplicates_pr_title:
-                        message = None
-                elif len(lines) == 1:
+                    continue
+                assert len(lines) > 0, "missing message content"
+                if len(lines) == 1:
```

Comment:
shall we `lines.pop(0)` here? otherwise we would be looking at the first line. and it should be empty.